app/src/main/java/com/eis/geoCalendar/app/GenericEvent.java

```diff
@@ -45,4 +47,17 @@ public T getContent() {
     public GPSPosition getPosition() {
         return position;
     }
+
+    /**
+     * Method to check whether two instances of {@link GenericEvent} are equal or not.
+     * @param obj The other Object to compare.
+     * @return {@code true} if the two objects are equal, {@code false} otherwise.
+     */
+    @Override
+    public boolean equals(@Nullable Object obj) {
+        if(!(obj instanceof GenericEvent))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['if' is not followed by whitespace.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4822079)